main_test.go

```diff
@@ -206,7 +206,8 @@ const emptyConfig = `
     ],
     "ForwardTo":[
     ],
-    "StatsDelay": "5s"
+    "StatsDelay": "5s",
+	"InternalMetricsReportingDelay": "5s"
```

Comment:
i would like to see a test where only the expected datapoints come out of that reporter. and they  are of the appropriate prefix, etc.

Reply:
working on it